### ISSUE TITLE
Error when deleting a photo from files: Division by zero

### ISSUE THREAD

**Issue by inthreedee** [pulsejet/memories#436]
I was testing out the memories app, connected to my `/Photos` directory. I noticed an error showing up in my logs whenever a photo was deleting outside the assigned `/Photos` directory:
`OC\DB\Exceptions\DbalException: An exception occurred while executing a query: SQLSTATE[22012]: Division by zero: 1365 Division by 0`

Repro:
1. Configure Memories to use /Photos
2. Upload a jpg to the root files directory /
3. Delete the jpg
4. Check logs

Memories app:  4.11.0
Nextcloud Server: 25.0.3snap3

Full log for this event:
```
{"reqId":"itOMXN3xeIljRirCPl1M","level":3,"time":"2023-02-23T16:03:54+00:00","remoteAddr":"172.20.0.7","user":"jonathan","app":"no app in context","method":"DELETE","url":"/remote.php/dav/files/jonathan/notIMG_2254.jpg","message":"An exception occurred while executing a query: SQLSTATE[22012]: Division by zero: 1365 Division by 0","userAgent":"Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/110.0","version":"25.0.3.2","exception":{"Exception":"OC\\DB\\Exceptions\\DbalException","Message":"An exception occurred while executing a query: SQLSTATE[22012]: Division by zero: 1365 Division by 0","Code":1365,"Trace":[{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":329,"function":"wrap","class":"OC\\DB\\Exceptions\\DbalException","type":"::"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":166,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":145,"function":"mapUpdateAggregates","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWrite.php","line":257,"function":"mapRemoveFromCluster","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Listeners/PostDeleteListener.php","line":51,"function":"deleteFile","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/ServiceEventListener.php","line":87,"function":"handle","class":"OCA\\Memories\\Listeners\\PostDeleteListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":251,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":73,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":88,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":100,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/Node/HookConnector.php","line":156,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/legacy/OC_Hook.php","line":106,"function":"postDelete","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1294,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1222,"function":"runHooks","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":727,"function":"basicOperation","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Connector/Sabre/File.php","line":522,"function":"unlink","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Tree.php","line":179,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Server.php","line":360,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/remote.php","line":171,"args":["/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/snap/nextcloud/33738/htdocs/lib/private/DB/Exceptions/DbalException.php","Line":71,"Previous":{"Exception":"Doctrine\\DBAL\\Exception\\DriverException","Message":"An exception occurred while executing a query: SQLSTATE[22012]: Division by zero: 1365 Division by 0","Code":1365,"Trace":[{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Connection.php","line":1780,"function":"convert","class":"Doctrine\\DBAL\\Driver\\API\\MySQL\\ExceptionConverter","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Connection.php","line":1719,"function":"handleDriverException","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Connection.php","line":1173,"function":"convertExceptionDuringQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":354,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":281,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":327,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":166,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":145,"function":"mapUpdateAggregates","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWrite.php","line":257,"function":"mapRemoveFromCluster","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Listeners/PostDeleteListener.php","line":51,"function":"deleteFile","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/ServiceEventListener.php","line":87,"function":"handle","class":"OCA\\Memories\\Listeners\\PostDeleteListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":251,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":73,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":88,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":100,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/Node/HookConnector.php","line":156,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/legacy/OC_Hook.php","line":106,"function":"postDelete","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1294,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1222,"function":"runHooks","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":727,"function":"basicOperation","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Connector/Sabre/File.php","line":522,"function":"unlink","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Tree.php","line":179,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Server.php","line":360,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/remote.php","line":171,"args":["/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php","Line":114,"Previous":{"Exception":"Doctrine\\DBAL\\Driver\\PDO\\Exception","Message":"SQLSTATE[22012]: Division by zero: 1365 Division by 0","Code":1365,"Trace":[{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":94,"function":"new","class":"Doctrine\\DBAL\\Driver\\PDO\\Exception","type":"::"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Connection.php","line":1163,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":354,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":281,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":327,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":166,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":145,"function":"mapUpdateAggregates","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWrite.php","line":257,"function":"mapRemoveFromCluster","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Listeners/PostDeleteListener.php","line":51,"function":"deleteFile","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/ServiceEventListener.php","line":87,"function":"handle","class":"OCA\\Memories\\Listeners\\PostDeleteListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":251,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":73,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":88,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":100,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/Node/HookConnector.php","line":156,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/legacy/OC_Hook.php","line":106,"function":"postDelete","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1294,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1222,"function":"runHooks","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":727,"function":"basicOperation","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Connector/Sabre/File.php","line":522,"function":"unlink","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Tree.php","line":179,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Server.php","line":360,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/remote.php","line":171,"args":["/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Driver/PDO/Exception.php","Line":26,"Previous":{"Exception":"PDOException","Message":"SQLSTATE[22012]: Division by zero: 1365 Division by 0","Code":"22012","Trace":[{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","line":92,"function":"execute","class":"PDOStatement","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Connection.php","line":1163,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDO\\Statement","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/Connection.php","line":294,"function":"executeStatement","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php","line":354,"function":"executeStatement","class":"OC\\DB\\Connection","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":281,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/DB/QueryBuilder/QueryBuilder.php","line":327,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":166,"function":"executeStatement","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWriteMap.php","line":145,"function":"mapUpdateAggregates","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Db/TimelineWrite.php","line":257,"function":"mapRemoveFromCluster","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/var/snap/nextcloud/33738/nextcloud/extra-apps/memories/lib/Listeners/PostDeleteListener.php","line":51,"function":"deleteFile","class":"OCA\\Memories\\Db\\TimelineWrite","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/ServiceEventListener.php","line":87,"function":"handle","class":"OCA\\Memories\\Listeners\\PostDeleteListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":251,"function":"__invoke","class":"OC\\EventDispatcher\\ServiceEventListener","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/symfony/event-dispatcher/EventDispatcher.php","line":73,"function":"callListeners","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":88,"function":"dispatch","class":"Symfony\\Component\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/EventDispatcher/EventDispatcher.php","line":100,"function":"dispatch","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/Node/HookConnector.php","line":156,"function":"dispatchTyped","class":"OC\\EventDispatcher\\EventDispatcher","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/legacy/OC_Hook.php","line":106,"function":"postDelete","class":"OC\\Files\\Node\\HookConnector","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1294,"function":"emit","class":"OC_Hook","type":"::"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":1222,"function":"runHooks","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/lib/private/Files/View.php","line":727,"function":"basicOperation","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Connector/Sabre/File.php","line":522,"function":"unlink","class":"OC\\Files\\View","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Tree.php","line":179,"function":"delete","class":"OCA\\DAV\\Connector\\Sabre\\File","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/CorePlugin.php","line":281,"function":"delete","class":"Sabre\\DAV\\Tree","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/event/lib/WildcardEmitterTrait.php","line":89,"function":"httpDelete","class":"Sabre\\DAV\\CorePlugin","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":472,"function":"emit","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":253,"function":"invokeMethod","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/3rdparty/sabre/dav/lib/DAV/Server.php","line":321,"function":"start","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/lib/Server.php","line":360,"function":"exec","class":"Sabre\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php","line":35,"function":"exec","class":"OCA\\DAV\\Server","type":"->"},{"file":"/snap/nextcloud/33738/htdocs/remote.php","line":171,"args":["/snap/nextcloud/33738/htdocs/apps/dav/appinfo/v2/remote.php"],"function":"require_once"}],"File":"/snap/nextcloud/33738/htdocs/3rdparty/doctrine/dbal/src/Driver/PDO/Statement.php","Line":92}}},"CustomMessage":"--"},"id":"63f78e6b0e0c9"}
```

**Comment by meichthys** [comment]
I can't seem to reproduce this. 
@inthreedee How are you deleting the jpg? Are you deleting it from within Memories after running `occ memories:idex`? or are you deleting it via the Nextcloud Files app? I can't seem to get the error either way.
If you're deleting from within Memories, are you deleting it from the default Timeline view, or a different view like the Folder view?

I do have the same structure as you (Photos are in /Photos). I tried uploading photo to root directory, then indexing, then deleting the photo from Memories

**Comment by inthreedee** [comment]
This happens to me when deleting a photo using the Nextcloud Files app, not Memories.

The photo does not appear in the Memories app, as it's only pointing to /Photos and the test photo is in the files root. It happens whether I delete the file using the Nextcloud web interface or from my local filesystem, which the sync client then picks up and handles.

Some extra info in case it matters: I had installed the Memories app and put a dozen or so photos in various folders into /Photos as a test, and then did nothing further with it. So I was not intentionally indexing photos or managing photos in Memories at all. It was just sitting there waiting for me to go back and look at it some more. I only noticed this error while managing photos outside of the Memories app. Disabling memories makes the error go away.
